packages/types/src/schemas/i18n.ts

```diff
@@ -66,6 +66,16 @@ export const i18nSchema = z.object({
    */
   defaultPreviewTitleText: z.string().optional().describe('Text shown on top of the preview section.'),
 
+  /**
+   * Title or tooltip shown when hovering the reload icon for a preview.
```

Comment:
Consider: I would like to avoid using the word `tooltip` here, or mention anything about hovering. Maybe something like:

```suggestion
   * Title attribute for the preview reload button.
```